README.md

```diff
@@ -1,4 +1,10 @@
 ![Tracee Logo](images/tracee.png)
+
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/aquasecurity/tracee)](https://github.com/aquasecurity/tracee/releases)
+[![Go Report Card](https://goreportcard.com/badge/github.com/aquasecurity/tracee)](https://goreportcard.com/report/github.com/aquasecurity/tracee)
+[![GitHub](https://img.shields.io/github/license/aquasecurity/tracee)](https://github.com/aquasecurity/tracee/blob/master/LICENSE)
```

Comment:
just rename the alt here from GitHub to License and I'll approve

Reply:
Done